### ISSUE TITLE
There is no way to install this extension from extensions.gnome.org and option gui are inacesible in ubuntu 21.04

### ISSUE THREAD

**Issue by M3t4ll1k0** [alireza6677/lyrics-finder-gnome-ext#14]
As i commented on the site there is not a version for gnome and no way to install it from web broser in ubuntu 21.04 and installing it with the instructions that tell to clone the git move the files to the local extension makes it able to install but options are un-accessible  her the error log and error screen:

The settings of extension LyricsFinder@alireza6677.gmail.com had an error:
```
Gtk.BuilderError: /home/metaliko/.local/share/gnome-shell/extensions/LyricsFinder@alireza6677.gmail.com/prefs.ui:107:1 Unhandled tag: <layout>
```

Stack trace:
```
buildPrefsWidget@/home/metaliko/.local/share/gnome-shell/extensions/LyricsFinder@alireza6677.gmail.com/prefs.js:18:13
_init@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:207:40
OpenExtensionPrefsAsync/<@resource:///org/gnome/Shell/Extensions/js/extensionsService.js:122:28
asyncCallback@resource:///org/gnome/gjs/modules/core/overrides/Gio.js:115:22
run@resource:///org/gnome/Shell/Extensions/js/dbusService.js:177:20
main@resource:///org/gnome/Shell/Extensions/js/main.js:19:13
run@resource:///org/gnome/gjs/modules/script/package.js:206:19
start@resource:///org/gnome/gjs/modules/script/package.js:190:8
@/usr/share/gnome-shell/org.gnome.Shell.Extensions:1:17
```
![image](https://user-images.githubusercontent.com/88471429/129069669-2c3c6fef-1d41-488e-8f51-43c8de6c60ed.png)

**Comment by TheWeirdDev** [comment]
As I've mentioned [here](https://github.com/TheWeirdDev/lyrics-finder-gnome-ext/issues/12#issuecomment-716028304), Gnome 3.38 had a weird bug that disallowed me from releasing an update for it. Fortunately it was fixed in 40.0.
Ubuntu will get Gnome 40 soon and there is nothing I can do about that bug.

**Comment by M3t4ll1k0** [comment]
I apologice and explained myself on bug #15, keep up the good work, cant whait to have your extionion on my shell.
